lib/src/components/board/client.dart

```diff
@@ -31,17 +31,18 @@ class BoardClient extends Board {
   @override
   Future<void> setGpioState(String pin, bool high, {Map<String, dynamic>? extra}) async {
     await _client.setGPIO(SetGPIORequest(name: name, pin: pin, high: high, extra: extra?.toStruct()));
+    return;
```

Comment:
Why is this needed?

Reply:
oh good catch thank you! it is not needed. I added that while debugging and did not mean to include it. 🪓